nntrainer/layers/layer_devel.h

```diff
@@ -255,6 +265,15 @@ class Layer {
    */
   virtual bool supportInPlace() const { return is_inplace; }
 
+  /**
+   * @brief Get the inplace direction for the tensor operation layer
+   *
+   * @return InPlaceDirection
+   */
+  virtual InPlaceDirection getInPlaceDirection() {
+    return InPlaceDirection::NONE;
```

Comment:
Even though it is virtual, is it okay to return `InPlaceDirection::NONE` regardless of the `is_inplace`?
According to the meaning of the `InPlaceDirection::NONE`, it holds only when `is_inplace = false`.

Reply:
As discussed offline, the `InPlaceDirection` option has meaning only when the layer type is `binary`. Therefore, even if the `is_inplace` option is set to `true`, it may return `NONE` if the layer type is not `binary`.